### ISSUE TITLE
iRacing: Wrong SessionTypeName in Qualy

### ISSUE THREAD

**Issue by hieblmedia** [zegreatclan/SimHub#365]
Hello,
the property "DataCorePlugin.GameData.NewData.SessionTypeName" is Race in the Qualifing.
Replay: https://drive.google.com/file/d/1zMj-DAmNlN6VNXDCtASrREaItXl_kCSl/view?usp=sharing

**Comment by zegreatclan** [comment]
Hi ! 
I've checked your replay but I could not see the issue, the expected value was showing : 
![image](https://user-images.githubusercontent.com/2207331/60400451-232a5800-9b74-11e9-855e-cc6d9e70f9da.png)

I also checked my code it transfers directly the following property coming from the game : SessionData.WeekendInfo.EventType with no additional post processing which must be pretty safe

Is it possible that it was used in a wider formula which was crashing ? In this case the result won't show and get stuck on the last valid value probably race in your case. 

Let me know, 

Nicolas

**Comment by Romainrob** [comment]
Hi! I hope you won't mind if I jump in.
I noticed this issue too and I believe this could be an iRacing thing.
When you're in a race session with warmup and quali attached, The property value is "Race" during the warmup and quali.
If the property directly displays the game data, I guess this is how the iRacing temetry works.

**Comment by hieblmedia** [comment]
@zegreatclan Your screenshot is from rFactor.
This issue is only in iRacing. I made a property with my own plugin for an fast fix with following lines.
Works also on viewing replays for me.

```
iRacingSDK.SessionData._SessionInfo._Sessions currentSession = ((IEnumerable<iRacingSDK.SessionData._SessionInfo._Sessions>)gameData.GameNewData.Raw.SessionData.SessionInfo.Sessions).SingleOrDefault(s => s.SessionNum == gameData.GameNewData.Raw.Telemetry.SessionNum);
if (currentSession != null)
{
    pluginManager.SetPropertyValue("SessionTypeName", OverTheClockDataPlugin.StaticType, (object)currentSession.SessionType);
}
```

**Comment by zegreatclan** [comment]
Dammit ! I will double check, I think this heat wave got me exhausted. Sorry for this confusion.

**Comment by zegreatclan** [comment]
Thanks again for the infos, i've fixed it !

**Comment by zegreatclan** [comment]
Fix released with simhub 6.8.2 ;) Thanks again for the report !
